src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,158 @@
+<h2>DESCRIPTION</h2>  <em>r.survey</em> is aimed at earth surface 
```

Comment:
Here the entire formatting is lost, why is that? Please...

Reply:
Hi Markus.
Sorry. Clearly we erroneously interpreted a comment about reducing the length of the strings with Geany.
Now the html file is formatted and the examples of usage of r.survey are correctly displayed in a single line.